src/IO/WriteHelpers.h

```diff
@@ -811,17 +811,25 @@ void writeIPv4Text(const IPv4 & ip, WriteBuffer & buf);
 void writeIPv6Text(const IPv6 & ip, WriteBuffer & buf);
 
 template <typename DecimalType>
```

Comment:
put trim_suffix_zeros as template arguments to avoid influence  the performance of  old codes  

template <bool flag = false>
void foo()
{
    original codes;
    if constexpr (flag)
    {
         new codes;
    }
}

Reply:
Thank you for the reminder. New code can be optimized at compile time.